docs/documentation/mission_module/extensibility/flight_point.rst

```diff
@@ -64,15 +63,20 @@ smoothly, especially when exchanging data with pandas, you have to work at class
 This can be done using :meth:`~fastoad.model_base.flight_point.FlightPoint.add_field`, preferably
 outside of any class or function::
```

Comment:
I suggest removing the "of" on that line to be consistent with the description of the FlightPoint dataclass

Reply:
Agreed. Done in 768e63f252a43106cc4e8ef0d321f3530fa44623.